### PR TITLE
Run `rubocop -a` on the monitor, downtime, and SLO code examples, and updated some snippets.

### PR DESCRIPTION
**Pull request by armcburney**
### What does this PR do?
- Formats all monitor, downtime, and service level objective Ruby code using the `dogapi-rb` client library to be uniform and consistent.
- Updates the comments on some examples to make more sense.

### Preview link
https://docs-staging.datadoghq.com/armcburney/rubocop_monitors/api/?lang=ruby#monitors

### CODE REVIEW

File: content/en/api/monitors/code_snippets/api-monitor-delete.rb [hunk 1]
```diff
@@ -7,4 +6,4 @@
 dog = Dogapi::Client.new(api_key, app_key)
 
 # Delete a monitor
-dog.delete_monitor(62625)
\ No newline at end of file
+dog.delete_monitor(62_625)
```
Comment (athap) [inline content/en/api/monitors/code_snippets/api-monitor-delete.rb]: If find this slightly confusing with `_` 
Is this a ruby way?

File: content/en/api/monitors/code_snippets/api-monitor-edit.rb [hunk 2]
```diff
@@ -7,4 +6,9 @@
 dog = Dogapi::Client.new(api_key, app_key)
 
 # Edit an existing monitor
-dog.update_monitor(91879, "avg(last_5m):sum:system.net.bytes_rcvd{host:host0} > 100", :message => "Bytes received on host0", :name => "We may need to add web hosts if this is consistently high.")
\ No newline at end of file
+dog.update_monitor(
+  91_879,
```
Comment (athap) [inline content/en/api/monitors/code_snippets/api-monitor-edit.rb]: same as before

File: content/en/api/monitors/code_snippets/api-monitor-delete.rb (hunk 1 above)
Comment (armcburney) [reply content/en/api/monitors/code_snippets/api-monitor-delete.rb]: Yeah, this is the idiomatic Ruby way for long numbers:

- https://docs.ruby-lang.org/en/2.0.0/syntax/literals_rdoc.html#label-Numbers
- https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/NumericLiterals

Comment (armcburney) [reply content/en/api/monitors/code_snippets/api-monitor-delete.rb]: I can disable it in rubocop if you want - and then we can run the autoformatter to remove the underscores 🙏

Comment (athap) [reply content/en/api/monitors/code_snippets/api-monitor-delete.rb]: no all good!

**Pull request merged**